corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -1127,13 +1107,6 @@ def get_latest_owner_id(case_id, action=None):
                     self.last_submitted = datetime.utcnow()
                     self.rev_before_last_submitted = self._rev
                     self.save()
-                    if case_list:
-                        try:
-                            self.invalidate_cached_payloads()
```

Comment:
Im probably missing something simple but why do we not need to cache bust here anymore.